coriolisclient/base.py

```diff
@@ -135,9 +135,6 @@ def get(self):
     def __eq__(self, other):
         if not isinstance(other, Resource):
             return NotImplemented
-        # two resources of different types are not equal
-        if not isinstance(other, self.__class__):
-            return False
```

Comment:
Please add this code back. This equality is important when comparing resources of different types. (ex. Replica and Migration, these will not be equal, even if their info are the same).

Reply:
Also, please add this case when testing this method.